spec/supports/macros.rb

```diff
@@ -1,9 +1,5 @@
 # frozen_string_literal: true
 
-def sign_in(user)
-  post user_session_url, params: { user: { email: user.email, password: user.password } }
-end
-
```

Comment:
this i removed since its provide by helper devices

Reply:
Nice! We did not have this previously so I added it. Thanks for the info

Reply:
hehe i am becoming pro now haha